<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>530</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Центр управления сообщениями</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.934, y1:1, x2:0.127, y2:0.244, stop:0 rgba(10, 13, 18, 255), stop:1 rgba(54, 63, 80, 255));
font-family: Grammatica;
</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="input_form">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>581</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,20);
border: 1px solid rgba(255,255,255,40);
border-radius: 7;
color: white;
QPushButton {
background-color: rgba(255,255,255,30);
}

QPushButton:hover {
background-color: rgba(255,255,255,50);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
    </property>
    <widget class="QTextEdit" name="name">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,0);
border: 1px solid rgba(255,255,255,0);
border-radius: 7;
font: 11pt;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Grammatica'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Gramatica';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="number">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,0);
border: 1px solid rgba(255,255,255,0);
border-radius: 7;
font: 11pt;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Grammatica'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Gramatica';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_contact">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
font: 11pt;
background-color: rgba(255,255,255,20);
}

QPushButton:hover {
background-color: rgba(255,255,255,50);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
     </property>
     <property name="text">
      <string>Добавить контакт</string>
     </property>
    </widget>
    <widget class="QLabel" name="display_name">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Имя: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="display_number">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Телефон: +7&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="contacts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>191</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="display_search">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Поиск :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="arguments">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,0);
border: 1px solid rgba(255,255,255,0);
border-radius: 7;
font: 11pt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
background-color: rgba(255,255,255,0);
}

QPushButton:hover {
background-color: rgba(255,255,255,50);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/search.svg</normaloff>icons/search.svg</iconset>
     </property>
    </widget>
    <widget class="QTextBrowser" name="console">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>361</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Grammatica'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settings">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(255,255,255,20);
}

QPushButton:hover {
background-color: rgba(255,255,255,50);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
     </property>
     <property name="text">
      <string>Настройки</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/settings.svg</normaloff>icons/settings.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="open_folder">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(255,255,255,20);
}

QPushButton:hover {
background-color: rgba(255,255,255,50);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
     </property>
     <property name="text">
      <string>Файлы</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/folder.svg</normaloff>icons/folder.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="analyze">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(255,255,255,20);
}

QPushButton:hover {
background-color: rgba(255,255,255,50);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
     </property>
     <property name="text">
      <string>Анализировать данные</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/analyze.svg</normaloff>icons/analyze.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="get_messages">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(255,255,255,20);
}

QPushButton:hover {
background-color: rgba(255,255,255,50);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
     </property>
     <property name="text">
      <string>Получить сообщения</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/mail.svg</normaloff>icons/mail.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="restart">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(255,255,255,20);
}

QPushButton:hover {
background-color: rgba(255,255,255,50);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
     </property>
     <property name="text">
      <string>Перезагрузить модем</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/refresh.svg</normaloff>icons/refresh.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(255,255,255,20);
}

QPushButton:hover {
background-color: rgba(255,255,255,50);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/save.svg</normaloff>icons/save.svg</iconset>
     </property>
    </widget>
    <zorder>add_contact</zorder>
    <zorder>display_name</zorder>
    <zorder>display_number</zorder>
    <zorder>contacts</zorder>
    <zorder>display_search</zorder>
    <zorder>name</zorder>
    <zorder>number</zorder>
    <zorder>arguments</zorder>
    <zorder>search</zorder>
    <zorder>console</zorder>
    <zorder>settings</zorder>
    <zorder>open_folder</zorder>
    <zorder>analyze</zorder>
    <zorder>get_messages</zorder>
    <zorder>restart</zorder>
    <zorder>save</zorder>
   </widget>
   <widget class="QFrame" name="output_form">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>581</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,20);
border: 1px solid rgba(255,255,255,40);
border-radius: 7;
color: white;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1" rowspan="2">
      <widget class="QPushButton" name="sendButton">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>10000</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgba(255,255,255,30);
}

QPushButton:hover {
background-color: rgba(255,255,255,50);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
       </property>
       <property name="text">
        <string>Отправить!</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/bolt.svg</normaloff>icons/bolt.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0" rowspan="2">
      <widget class="QTextEdit" name="message"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="gridFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,20);
border: 1px solid rgba(255,255,255,40);
border-radius: 7;
color: white;
QPushButton {
background-color: rgba(255,255,255,30);
}

QPushButton:hover {
background-color: rgba(255,255,255,50);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
    </property>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(255,255,255,20);
}

QPushButton:hover {
background-color: rgba(255,10,10,80);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/close.svg</normaloff>icons/close.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="minimizeButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>0</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgba(255,255,255,20);
}

QPushButton:hover {
background-color: rgba(30,30,255,80);
}

QPushButton:pressed {
background-color: rgba(255,255,255,80);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/minimize.svg</normaloff>icons/minimize.svg</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="labeltitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: None;
background-color: rgba(255,255,255,0);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Центр управления сообщениями&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
